OpenKJ/main.cpp

```diff
@@ -106,16 +107,17 @@ int main(int argc, char *argv[])
 //    file.close();
 //    stylesheet = "* { background: #191919; color: #DDDDDD; border: 1px solid #5A5A5A;}";
 //    a.setStyleSheet(stylesheet);
-    MainWindow w;
```

Comment:
I'm assuming this line was accidentally removed along with the old single instance stuff.  It needs to be added back, but can be added directly before w.show(); is called after the single instance check.  This is breaking compilation since the MainWindow object is never created.